src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static int MAX_TASK = 100;
+    static Task[] tasks = new Task[MAX_TASK];
+    public static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printHello();
+        handleCommand();
+        printGoodbye();
+    }
+
+    private static void handleCommand() {
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            line = in.nextLine();
+            CommandType commandType = readCommandType(line);
+
+            switch(commandType) {
+            case COMMAND_BYE:
+                break;
+            case COMMAND_LIST:
+                printTasks();
+                break;
+            case COMMAND_DONE:
+                markAsDone(line);
+                break;
+            case COMMAND_TODO:
+                addTodo(line);
+                break;
+            case COMMAND_DEADLINE:
+                addDeadline(line);
+                break;
+            case COMMAND_EVENT:
+                addEvent(line);
+                break;
+            case COMMAND_UNKNOWN:
+                System.out.println("Unknown command. Type \"help\" for more information");
+                break;
+            }
+
+            if (commandType == CommandType.COMMAND_BYE) {
+                break;
+            }
+        }
+    }
+
+    private static void addTodo(String line) {
+        int i = line.indexOf(' ');
```

Comment:
Could the variable be changed from an iterator i to a variable name (e.g. spaceIndex). I noticed it in addDeadline() and addEvent() too.